src/ui/HeaderData1.svelte

```diff
@@ -5,7 +5,7 @@
 
 <header class="ons-header ons-header--hero" role="banner">
   <div class="ons-header__main ons-header__main--with-description">
-    <div class="ons-container">
+    <div class="ons-container" id="ons-container">
```

Comment:
I'd suggest to not use `ons-` prefixes on our stuff as it might bites us later, but let's leave that one in.